book/website/community-handbook/accessibility/accessibility-comms-platforms.md

```diff
@@ -0,0 +1,21 @@
+(ch-accessibility-comms-platforms)=
+# Access-centered support on Communication Platforms
+
+_The Turing Way_ uses a variety of platforms for communication (i.e. Slack, social media platforms like X/Twitter, Linkedin, Mastodon) and collaboration (i.e. Github, HackMD, Etherpad). 
```

Comment:
I think we try to avoid Latin abbreviations for readability and compatibility with screen readers.

Reply:
Great catch @JimMadge, thank you!

Reply:
Tried to correct this change ages ago - but I think it didn't work properly - editing this phrasing to 'such as...'